flipperdevices/flipperzero-firmware
# [FL-2692, FL-2604, FL-2632] New first start sequence

# What's new

- [FL-2692] desktop: restored automatic power off & manual power off on slideshow view; 
- [FL-2604, FL-2632] assets: added frames for new first start sequence; 
- docs: added info on slideshow compilation

# Verification 

- Build slideshow file with `python3 scripts/slideshow.py -i assets/slideshow/first_start/ -o assets/slideshow/first_start/slideshow`
- Upload that file to Flipper's internal storage
- Restart Flipper & validate both shutdown modes are available: long OK press & 1 hour idle
- Go through the slideshow and check for any visual issues

# Checklist (For Reviewer)



[FL-2692]: https://flipperzero.atlassian.net/browse/FL-2692?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
